modules/kubernetes-addons/prometheus/locals.tf

```diff
@@ -3,7 +3,7 @@ locals {
     name                       = "prometheus"
     chart                      = "prometheus"
     repository                 = "https://prometheus-community.github.io/helm-charts"
-    version                    = "14.6.0"
+    version                    = "15.3.0"
```

Comment:
@naris-silpakit Could you please find the all the occurrences of the Prometheus version and replace it? Some of the examples are also using the older version of Prometheus.

Reply:
Yep! I'll do that and push an update.

Reply:
@naris-silpakit please mark this as resolved once you have addressed @vara-bonthu's comment.